lib/cli/kit/config.rb

```diff
@@ -27,6 +27,18 @@ def get(section, name)
         all_configs.dig("[#{section}]", name) || false
       end
 
+      # Coalesce and enforce the value of a config to a boolean
+      def get_bool(section, name)
+        case get(section, name).to_s
```

Comment:
should we downcase?

Reply:
I'd rather it be explicitly `true` or `false` if we're going to start trending toward types.

Reply:
I thought about it, but this enforces consistency in the config files, so I’m ok either way.

Reply:
Im ok either way as well